lib/apiservers/engine/backends/container.go

```diff
@@ -64,32 +63,32 @@ const (
 
 // ContainerExecCreate sets up an exec in a running container.
 func (c *Container) ContainerExecCreate(config *types.ExecConfig) (string, error) {
-	return "", fmt.Errorf("%s does not implement container.ContainerExecCreate", c.ProductName)
+	return "", fmt.Errorf("%s does not implement container.ContainerExecCreate", ProductName())
```

Comment:
#1099.  It won't take long, but I do have another higher priority issues to work on.  Also, if we're buying into the small commits regime, then we should have the strings be another PR.  It's on my list of items to work on immediately after our TP3 work.